src/xdp/program.c

```diff
@@ -125,6 +160,89 @@ XdpInitializeFrameCache(
     Cache->Flags = 0;
 }
 
+static
+_Must_inspect_result_
+ebpf_result_t
+EbpfExtensionInvokeProgram(
+    _In_ const EBPF_EXTENSION_CLIENT *Client,
+    _In_ const VOID *Context,
+    _Out_ uint32_t *Result)
+{
+    ebpf_invoke_program_function_t InvokeProgram = Client->InvokeProgram;
+    const void* ClientBindingContext = Client->ClientBindingContext;
+
+    return InvokeProgram(ClientBindingContext, Context, Result);
+}
+
+static
+XDP_RX_ACTION
+XdpInvokeEbpf(
+    _In_ HANDLE EbpfTarget,
+    _In_ XDP_FRAME *Frame,
+    _In_opt_ XDP_RING *FragmentRing,
+    _In_opt_ XDP_EXTENSION *FragmentExtension,
+    _In_ UINT32 FragmentIndex,
+    _In_ XDP_EXTENSION *VirtualAddressExtension
+    )
```

Comment:
nit: inconsistent coding style between line 169 vs lines 185-186.  Consider using `clang-format` in the xdp-for-windows repo to automate this sort of thing.